conformance/system_symbols.ion

```diff
@@ -0,0 +1,18 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+(ion_1_x
+  (toplevel '#$1' '#$2' '#$3' '#$4' '#$5' '#$6' '#$7' '#$8' '#$9')
+  (produces $ion $ion_1_0
+            $ion_symbol_table name version imports symbols max_id
+            $ion_shared_symbol_table))
+
+(ion_1_0
+  (toplevel '#$10')
+  (signals "Invalid symbol address: 10"))
+
+(ion_1_1
+  (toplevel '#$10')
+  (produces $ion_literal))
```

Comment:
I don't recall seeing this in the spec. Can we put a "todo" here until the system module is defined?

Reply:
https://github.com/amazon-ion/ion-docs/blob/ion-11-specification/src/whatsnew.adoc#system-symbol-table-changes

Though I got the address here wrong, I'll fix that.

I'd be shocked if there's any debate whether `$ion_literal` and `$ion_encoding` are included, do you think there will be debate over their specific addresses?

Reply:
I see that now.

Can you remind me what `$ion_literal` is for?

Reply:
`$ion_literal` is how one treats as application data something looking like an encoding directive.

At top-level `$ion_literal::$ion_encoding::()` expands to `$ion_encoding::()` rather than being evaluated as a directive.  This avoids holes in the data model by ensuring anything can be written at top level.

It's a gap in the spec, and I know some edge cases we need to iron out, so I'll cut an issue.